example/alg_test.cpp

```diff
@@ -1,8 +1,8 @@
 #define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
 
 #include <iostream>
-#include "catch.hpp"
-#include <hsm/hsm.hpp>
+#include "catch2/catch.hpp"
+#include "hsm/hsm.hpp"
```

Comment:
keep the < here
From the perspective of the `example` code hsm and now with the external catch2 are libraries in use..